packages/manager/modules/dedicated-universe-components/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ovh-ux/ng-ovh-dedicated-universe-components",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud DedicatedUniverseComponents",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/dedicated-universe-components"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/ng-ovh-dedicated-universe-components' --include-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/ng-ovh-dedicated-universe-components' --include-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/ng-ovh-dedicated-universe-components' --include-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "@ovh-ux/ui-kit": "^4.2.3",
```

Comment:
are you sure those dependencies are used ?

Reply:
yes, oui-message component is used

Reply:
It should be a `peerDep`as it is an angular dependency